src/middlewared/middlewared/plugins/account.py

```diff
@@ -132,6 +132,22 @@ class Config:
         datastore_prefix = 'bsdusr_'
         cli_namespace = 'account.user'
 
+    # FIXME: Please see if dscache can potentially alter result(s) format, without ad, it doesn't seem to
+    RESULT_ENTRY = Patch(
+        'user_create', 'user_entry',
+        ('rm', {'name': 'group'}),
+        ('rm', {'name': 'group_create'}),
+        ('rm', {'name': 'home_mode'}),
+        ('rm', {'name': 'password'}),
+        ('add', Dict('group', additional_attrs=True)),
+        ('add', Int('id')),
+        ('add', Bool('builtin')),
+        ('add', Bool('id_type_both')),
+        ('add', Bool('local')),
+        ('add', Str('unixhash')),
+        ('add', Str('smbhash')),
```

Comment:
Directory services users don't return `unixhash` or `smbhash` for obvious reasons.

Reply:
@anodos325 that is good as they are not marked as `required`, i am although unsure if more fields can be added to the payload under certain flags/circumstances when ad is involved